.changeset/clean-eggs-accept.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/integration': patch
+---
+
+Enabled app-config to take apiVersion of azure integration to support azure devops of different APIs
```

Comment:
```suggestion
Enabled app-config to take `apiVersion` of azure integration to support azure devops of different APIs
```

and then add "apis" (all lowercase) to `.github/vale/config/vocabularies/Backstage/accept.txt` to make the spell checker happy.

Regarding wording, a user reading this in the changelog may want to know that this affects the three functions `getAzureCommitsUrl`, `getAzureDownloadUrl` and `getAzureFileFetchUrl` as well as the type `AzureIntegrationConfig` if they were using them.